spec/features/idv/analytics_spec.rb

```diff
@@ -584,7 +584,7 @@
         flow_path: 'standard', opted_in_to_in_person_proofing: false
       },
       'IdV: in person proofing state_id visited' => {
-        step: 'state_id', flow_path: 'standard', analytics_id: 'In Person Proofing', proofing_components: { document_check: 'usps' }
+        step: 'state_id', flow_path: 'standard', analytics_id: 'In Person Proofing'
```

Comment:
I want to call out this behavior change. `doc_auth_vendor` is now only set in the update method. IMHO this is correct behavior, because you don't really have a vendor until you've _done_ the thing. But if any old events are depending on this field being here, it won't be going forward.